lxd/db/cluster/certificates.mapper.go

```diff
@@ -194,33 +195,22 @@ func GetCertificateID(ctx context.Context, tx *sql.Tx, fingerprint string) (int6
 		return -1, fmt.Errorf("Failed to get \"certificateID\" prepared statement: %w", err)
 	}
 
-	rows, err := stmt.Query(fingerprint)
+	row := stmt.QueryRowContext(ctx, fingerprint)
+	err = row.Err()
```

Comment:
@masnax not a biggy, but according to https://cs.opensource.google/go/go/+/refs/tags/go1.19.1:src/database/sql/sql.go;l=3352-3353 its not necessary to check the `row.Error()` before calling `row.Scan()` as that will check the error first and return it. Would reduce the number of lines and function calls.